src/Provider/Nominatim/Nominatim.php

```diff
@@ -149,7 +154,6 @@ private function xmlResultToArray(\DOMElement $resultNode, \DOMElement $addressN
             }
         }
 
-        $builder->setPostalCode($postalCode);
```

Comment:
Why was this moved?

Reply:
Seemed more logical to regroup postal code process ;
But I really don't mind putting it back to line 152 ^_^